sdk/face/ai-vision-face-rest/README.md

```diff
@@ -0,0 +1,435 @@
+
+const endpoint = process.env['FACE_ENDPOINT'] || '<endpoint>';
+const credential = new DefaultAzureCredential();
+const client = createFaceClient(endpoint, credential);
+```
+
+## Key concepts
+
+### FaceClient
+
+The `FaceClient` is the primary interface for developers interacting with the Azure AI Face service. It follows the design of [REST client](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) and serves as the gateway from which all interaction with the library will occur.
+
+### Long-running operations
+
+Long-running operations are operations which consist of an initial request sent to the service to start an operation,
+followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has
+succeeded, to get the result.
+
+Methods that train a group (LargeFaceList, PersonGroup or LargePersonGroup), create/delete a Person/DynamicPersonGroup, 
+add a face or delete a face from a Person are modeled as long-running operations.
+
+The client exposes a `getLongRunningPoller` method that returns a poller object from the initial HTTP 202 response.
+Callers should wait for the operation to complete by calling `pollUntilDone()` on the poller object. Sample code snippets
+are provided to illustrate using long-running operations [below](#examples "Examples").
+
+## Examples
+
+The following section provides several code snippets covering some of the most common Face tasks, including:
+
+- [Detecting faces in an image](#face-detection "Face Detection")
+- [Identifing the specific face from a LargePersonGroup](#face-recognition-from-largepersongroup "Face Recognition from LargePersonGroup")
+- [Determining if a face in an video is real (live) or fake (spoof)](#liveness-detection "Liveness Detection")
+
+### Face Detection
+
+Detect faces and analyze them from an binary data.
+
+```js
+import { readFileSync } from 'fs';
+import { AzureKeyCredential } from '@azure/core-auth';
+
+import createFaceClient, {
+    Detect200Response,
+    FaceAttributeTypeDetection03,
+    FaceAttributeTypeRecognition04,
+} from '@azure-rest/ai-vision-face';
+
+const endpoint = process.env['FACE_ENDPOINT'] || '<endpoint>';
+const apikey = process.env['FACE_APIKEY'] || '<apikey>';
+const credential = new AzureKeyCredential(apikey);
+const client = createFaceClient(endpoint, credential);
+
+const response = await client.path('/detect').post({
+    contentType: 'application/octet-stream',
+    queryParameters: {
+        detectionModel: 'detection_03',
+        recognitionModel: 'recognition_04',
+        returnFaceLandmarks: true,
+        returnRecognitionModel: true,
+        faceIdTimeToLive: 120,
+        returnFaceAttributes: [FaceAttributeTypeDetection03.HEAD_POSE, FaceAttributeTypeDetection03.MASK, FaceAttributeTypeRecognition04.QUALITY_FOR_RECOGNITION],
+        returnFaceId: false,
+    },
+    body: readFileSync('path/to/test/image'),
```

Comment:
Pls ensure the sample works and up-to-date. I believe the `FaceAttributeTypeRecognition04` is removed.